### PR TITLE
Adds an access requirement to supermatter shard crates

### PR DESCRIPTION
**Pull request by Sprok0**
The fancy black supermatter crates had a lock, but no access checks on them, so anyone can open them.
The supermatter crate now has access_engine_equip required to open it.
Makes the shard orderable from cargo also come in a shard crate, and changes it's access and name to match the existing shard crate.

### CODE REVIEW

**Comment by eplgr** [comment]
Can mechanics open them?
Also, I see no reason for the cargo shard having any different requirement.

**Comment by Sprok0** [comment]
The cargo shard comes in a normal crate that can't be re-closed or re-locked, so it needs to be opened more carefully. At least that's why I figure that it has higher requirements to unlock.

**Comment by Intigracy** [comment]
Pretty sure you can re-lock crates if you hit them with your ID while they're closed.
But honestly supermatter shards should come in the supermatter shard crate, the orderable one was in before the crate and it was never changed.

**Comment by Intigracy** [comment]
done?

**Comment by Sprok0** [comment]
should be done

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -1378,10 +1378,10 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	contains = list(/obj/machinery/power/supermatter/shard)
 	name = "Supermatter shard"
 	cost = 500 //So cargo thinks thrice before killing themselves with it. You're going to need a department account most likely.
-	containertype = /obj/structure/closet/crate/secure/engisec
-	containername = "supermatter shard crate"
+	containertype = /obj/structure/closet/crate/secure/large/reinforced/shard/empty
+	containername = "Supermatter Shard Crate"
```
Comment (Intigracy) [inline code/datums/supplypacks.dm]: "supermatter shard crate" so that it's in line with the others

Comment (Sprok0) [reply code/datums/supplypacks.dm]: that makes it out of line with the actual supermatter shard crates not from cargo

**Comment by mrpain666** [comment]
Pretty sure I was able to open one without ANY ID in hand but I'd have to double check that

File: code/datums/supplypacks.dm (hunk 1 above)
Comment (Intigracy) [reply code/datums/supplypacks.dm]: You can change those too!

All the ones in the supply file that I'm seeing are lowercase except for proper nouns, it was standardized a short while ago too.

**Comment by Intigracy** [comment]
You can before this PR, I was talking about re-locking.

**Comment by mrpain666** [comment]
Yeah. There really should be an access requirement on them. If a lock is opened by any key, the lock is pretty useless, isnt it?

Crate locks are sluts.

**Comment by Intigracy** [comment]
That's what this PR does you nerd.

**Pull request merged**